samples/install-without-bom/pom.xml

```diff
@@ -31,8 +31,12 @@
       <artifactId>google-cloud-document-ai</artifactId>
       <version>0.1.1</version>
     </dependency>
-    <!-- [END documentai_install_without_bom] -->
-
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>1.103.1</version>
+    </dependency>
+      <!-- [END documentai_install_without_bom] -->
```

Comment:
Please don't move this snippet unless everyone who uses document-ai must also use storage (then it should be a dependency). Otherwise, this will show up in the install instructions on the README and eventually cloudsite.